backend/fms_core/tests/valid_templates/Experiment_Infinium_24_v3_4_0_B_A_1.csv

```diff
@@ -1,6 +1,6 @@
 Experiment Submission Template,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
I think this file won't be used anymore. The tests are not made from the CSV files for the importers

Reply:
yeah i know but i changed it in case there was still stuff tied to it ... We need to do some cleanup ...